frontends/web/src/routes/account/actionButtons.tsx

```diff
@@ -61,17 +62,42 @@ export const ActionButtons = ({ canSend, code, coinCode, exchangeSupported, acco
           {t('button.send')}
         </span>
       )}
-      <Link key="receive" to={`/account/${code}/receive`} className={style.receive}>
-        <span>{t('button.receive')}</span>
-      </Link>
-      { exchangeSupported && (
-        <Link key="exchange" to={`/exchange/info/${code}`} className={style.exchange}>
-          <span>{t('generic.buySell')}</span>
+
+      {accountDataLoaded ? (
+        <Link key="receive" to={`/account/${code}/receive`} className={style.receive}>
+          <span>{t('button.receive')}</span>
         </Link>
+      ) : (
+        <span key="receiveDisabled" className={`${style.receive} ${style.disabled}`}>
+          {t('button.receive')}
+        </span>
       )}
-      {walletConnectEnabled && <Link key="wallet-connect" to={`/account/${code}/wallet-connect/dashboard`} className={style.walletConnect}>
-        <WalletConnectLight width={24}/> {!isLargeTablet && <span>Wallet Connect</span>}
-      </Link>}
+
+      {exchangeSupported && (
+        <>
+          {accountDataLoaded ?
+            <Link key="exchange" to={`/exchange/info/${code}`} className={style.exchange}>
+              <span>{t('generic.buySell')}</span>
+            </Link> :
```

Comment:
nit: I find this hard to read
please use `accountDataLoaded ? ( \newline ) : ( \newline ) ` here and in other places.. 

would it help if I try to find an eslint rule to force this?